spec/lib/tasks/update_super_group_ids_spec.rb

```diff
@@ -3,7 +3,7 @@
 Rake::Task.define_task(:environment)
 
 RSpec.describe 'Migrating carrier specific super group Id', :type => :task do
-  let(:plan) { FactoryGirl.build(:plan, hios_id: "88806MA0030001-01") }
+  let(:plan) { FactoryGirl.build(:plan, hios_id: "88806MA0030001-01", active_year: 2017) }
```

Comment:
still not sure why this needs to be locked per year

Reply:
rake `lib/tasks/update_super_group_ids.rake` had .xlsx file where the plan year is addressed as 2017. So the spec is mocked such a way to take 2017 as the active year